CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 ## v1210
 
-* Staging release date: TBD
-* Production APM-injected release date: TBD
-* Production Standalone release date: TBD
+* Staging release date: 06/22/2021
+* Production APM-injected release date: 07/06/2021
+* Production Standalone release date: 07/08/2021
```

Comment:
Typically we wait a week after APM-injected prod release before we release for copy/paste. I would leave the last date as TBD for now and update when we release it next week.